Source/StateA.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * This class is provided as an example of a concrete 
+ * implementation of ButtonState
+ */
+package cs505.group1.state;
+
+/**
+ *
+ * @author CS505 Fall17 Group1
+ * 
+ * TO CHANGE THE STATE:
+ * The line "ButtonStateA newState = new ButtonStateA();"
+ * in each method can be changed to a different state such as
+ * "ButtonStateA newState = new ButtonStateB();"
```

Comment:
Code is correct, update the comment to reflect the variable type is the more generic ButtonState to allow this switch of state.

Reply:
change made